department-for-transport-BODS/create-data
# feat: CFD-471 Master stop list

# Description

-   Add the master stop list for inbound and outbound journeys

# Testing instructions

-   Create a single and a return and check the matching allows you to use all stops for all journey patterns

# Type of change
